L1Trigger/Phase2L1ParticleFlow/src/newfirmware/jetmet/L1SeedConePFJetEmulator.h

```diff
@@ -0,0 +1,231 @@
+#ifndef L1SEEDCONEPFJETEMULATOR_H
+#define L1SEEDCONEPFJETEMULATOR_H
+
+#include "ap_int.h"
+#include "ap_fixed.h"
+#include "../dataformats/puppi.h"
+#include "../dataformats/jets.h"
+#include <iostream>
+#include <vector>
+#include <numeric>
+#include <algorithm>
+
+namespace L1SCJetEmu {
```

Comment:
This could become a class, with the Config passed in the constructor and the lookup table initialized only once in the constructor?